### ISSUE TITLE
Add support for signing images with cosign

### ISSUE THREAD

**Issue by axtloss** [Vanilla-OS/Vib#33]
View https://github.com/Vanilla-OS/ABRoot/issues/261 for more details

**Comment by xynydev** [comment]
This is something that needs to be done in CI / GitHub Actions after the container image is built from the Containerfile built by Vib.

**Comment by kbdharun** [comment]
> This is something that needs to be done in CI / GitHub Actions after the container image is built from the Containerfile built by Vib.

Yep, the sigstore's cosign action should be run only after pushing the image to the respective registry.

**Comment by axtloss** [comment]
That doesn't mean it's something vib can't handle.

**Comment by axtloss** [comment]
With #76 being merged this only requires a finalize plugin
